src/styles/blocks/card.scss

```
@@ -0,0 +1,121 @@
+.card {
+  //box-sizing: border-box;
+  background-color: $color-accent-white;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  place-self: center;
+  transition: transform 0.5s ease;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &:hover &__title {
+    color: $color-hover-title-blue;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    @include flexAlign;
+    margin-top: 16px;
+  }
+
+  &__rating-reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-main-accent;
+  }
+
+  &__prices {
+    @include flexAlign;
+    margin-top: 24px;
```

use parent selector
    https://sass-lang.com/documentation/style-rules/parent-selector/
```suggestion
    margin-top: 24px;
    
    &-text {},
    &-number {}

```